futaba/cogs/settings/core.py

```diff
@@ -342,3 +343,55 @@ def mention(role):
 
         await ctx.send(embed=embed)
         self.journal.send("roles/jail", ctx.guild, content, icon="settings", role=role)
+
+    @commands.command(name="trackblacklistadd")
+    @commands.guild_only()
+    @permissions.check_mod()
+    async def tracker_blacklist_add(
+        self, ctx, *, user_or_channel: Union[MemberConv, TextChannelConv]
+    ):
+        """ Add a user or channel to the tracking blacklist. """
+
+        logger.info(
+            "Adding '%s' (%d) to the tracking blacklist for guild '%s' (%d)",
+            user_or_channel.name,
+            user_or_channel.id,
+            ctx.guild.name,
+            ctx.guild.id,
+        )
+
+        with self.bot.sql.transaction():
+            self.bot.sql.settings.add_to_tracking_blacklist(ctx.guild, user_or_channel)
+
+        embed = discord.Embed(colour=discord.Colour.green())
```

Comment:
Should be `dark_teal()`. See [here](https://github.com/strinking/futaba/blob/master/CONTRIBUTING.md).